### PR TITLE
Update docs about subscription for tax webhooks

### PR DESCRIPTION
**Pull request by korycins**
Update docs, for PR: https://github.com/saleor/saleor/pull/10433

### CODE REVIEW

File: docs/developer/api-reference/objects/taxable-object-line.mdx [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+
+The source line related to this tax line.
+
+#### [`quantity`](#) ([`Int!`](../../../developer/api-reference/scalars/int))
+
+Number of items.
+
+#### [`chargeTaxes`](#) ([`Boolean!`](../../../developer/api-reference/scalars/boolean))
+
+Determine if taxes are being charged for the product.
```
Comment (pawelpikus) [inline docs/developer/api-reference/objects/taxable-object-line.mdx:34]:
```suggestion
Determines if taxes are being charged for the product.
```

File: docs/developer/api-reference/objects/taxable-object-discount.mdx [hunk 2]
```diff
@@ -0,0 +1,24 @@
+---
+id: taxable-object-discount
+title: TaxableObjectDiscount
+hide_table_of_contents: false
+---
+
+No description
```
Comment (pawelpikus) [inline docs/developer/api-reference/objects/taxable-object-discount.mdx:7]: Is this correct?

File: docs/developer/api-reference/objects/taxable-object-line.mdx [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+
+#### [`productName`](#) ([`String!`](../../../developer/api-reference/scalars/string))
+
+The product name.
+
+#### [`variantName`](#) ([`String!`](../../../developer/api-reference/scalars/string))
+
+The variant name
```
Comment (pawelpikus) [inline docs/developer/api-reference/objects/taxable-object-line.mdx:42]:
```suggestion
The variant name.
```

File: docs/developer/api-reference/objects/taxable-object.mdx [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+
+### Fields
+
+#### [`sourceObject`](#) ([`TaxSourceObject!`](../../../developer/api-reference/unions/tax-source-object))
+
+The source object related to this tax object.
+
+#### [`pricesEnteredWithTax`](#) ([`Boolean!`](../../../developer/api-reference/scalars/boolean))
+
+Determine if prices contain entered tax.
```
Comment (pawelpikus) [inline docs/developer/api-reference/objects/taxable-object.mdx:30]:
```suggestion
Determines if prices contain entered tax.
```

File: docs/developer/extending/apps/synchronous-webhooks/tax-webhooks.mdx [hunk 5]
```diff
@@ -76,9 +76,52 @@ For example, the amount of 12,34 USD will be sent as `"total_amount": "12.34"`
...
+        shippingPrice {
+          amount
+        }
+      }
+    }
+  }
+}
+```
+
+A payload sent to the app would look like this:
```
Comment (pawelpikus) [inline docs/developer/extending/apps/synchronous-webhooks/tax-webhooks.mdx:110]:
```suggestion
The payload sent to the app would look like this:
```

File: docs/developer/extending/apps/synchronous-webhooks/tax-webhooks.mdx [hunk 6]
```diff
@@ -230,9 +273,52 @@ For example, the amount of 12,34 USD will be sent as `"total_amount": "12.34"`
...
```
Comment (pawelpikus) [inline docs/developer/extending/apps/synchronous-webhooks/tax-webhooks.mdx:307]:
```suggestion
The payload sent to the app would look like this:
```

**Review by pawelpikus — APPROVED** [review]

**Review by pawelpikus — APPROVED** [review]

**Review by pawelpikus — APPROVED** [review]

**Comment by korycins** [comment]
I've applied all requested changes but I think that we need to wait to merge this PR until core changes will be merged: https://github.com/saleor/saleor/pull/10433

**Comment by peelar** [comment]
> I've applied all requested changes but I think that we need to wait to merge this PR until core changes will be merged: [saleor/saleor#10433](https://github.com/saleor/saleor/pull/10433)

Is there any chance you can ping me after it's merged? 🙏

**Comment by korycins** [comment]
> @korycins Is there any chance you can ping me after it's merged? 

@peelar  Yeah, sure

**Review by peelar — APPROVED** [review]

**Pull request merged**